### PR TITLE
Assigment4 

### PR DESCRIPTION
**Pull request by josellorentev**

### CODE REVIEW

File: Assignment4/josellorentev-190021/task06.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/105RNbZdDPid-La5HphhUvLXM3l9nYj8i
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/josellorentev-190021/task06.py]: won't work in Python. This is notebook magic

File: Assignment4/josellorentev-190021/task06.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+ns = Namespace("http://somewhere#")
+g.add((ns.Ump, RDF.type, ns.University))
+g.add((ns.JohnSmith, vcard.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/josellorentev-190021/task06.py:75]: work is not defined in vcard. Define a new property on your namespace

**Review by dgarijo — CHANGES_REQUESTED** [review]
Small changes requested

File: Assignment4/josellorentev-190021/task06.py (hunk 2 above)
Comment (josellorentev) [reply Assignment4/josellorentev-190021/task06.py:75]: i have done the changes already and i have done the first assigment of the proyect, i think that both merge in one pull request. Do i need close one and do two separately request or with this one it is correct?

Comment (dgarijo) [reply Assignment4/josellorentev-190021/task06.py:75]: should be ok.

Comment (dgarijo) [reply Assignment4/josellorentev-190021/task06.py:75]: This is still not done. You are still defining the work property as a property in vcard

**Comment by dgarijo** [comment]
I am merging this to avoid conflicts. However, Assignment 6 is not yet correct

**Pull request merged**